[Collection] - DeCyfir (partner pack) (#40892)
demisto/content: Packs/DeCYFIR/ReleaseNotes/1_0_19.md

> <~XSIAM>

Reviewer: Should this also be shown in the platform? Currently it is only XSIAM..

#### Integrations

##### New: DeCYFIR Event Collector

> - New: Added a new integration- DecyfirEventCollector that Collects event logs from Decyfir for ingestion into Cortex XSIAM.

Reviewer 2: ```suggestion
- New: Added the **DecyfirEventCollector** integration, which collects event logs from Decyfir for ingestion into Cortex XSIAM.
```
